azure-pipelines.yml

```diff
@@ -241,6 +242,14 @@ stages:
           workingDirectory: $(Build.SourcesDirectory)/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmDebuggingExtension
           failOnStderr: true
           condition: and(succeeded(), eq(variables['_BuildConfig'], 'Release'))
+        - task: PublishTestResults@2
```

Comment:
were we not publishing them before?

Reply:
We were publishing them, but changing to `dotnet test` changed the format they were emitted in so I had to react.